File: 백준/Bronze/2576. 홀수/홀수.cc
#include <iostream>
using namespace std;

int main() {

	int minN = 100;
	int sum = 0;

	for (int i = 0; i < 7; i++)
	{
		int n;
		cin >> n;

		if (n % 2 == 1)
		{
			sum += n;

			if (n <= minN)
			{
				minN = n;
			}
		}
	}

	if (minN == 100)
	{
		cout << -1 << endl;
		return 0;
	}

	cout << sum << "\n" << minN << endl;
}

File: 백준/Bronze/2576. 홀수/홀수.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        int sum = 0;
        int minN = 100;

        for (int i = 0; i < 7; i++) {
            int n = sc.nextInt();

            if (n % 2 == 1) {
                sum += n;

                if (n <= minN) {
                    minN = n;
                }
            }
        }

        if (minN == 100) {
            System.out.println(-1);
            return;
        }

        System.out.println(sum);
        System.out.println(minN);
    }
}